src/main/drivers/pwm_mapping.c

```diff
@@ -235,6 +235,13 @@ pwmOutputConfiguration_t *pwmInit(drv_pwm_config_t *init)
                 type = MAP_TO_SERVO_OUTPUT;
 #endif
 
+#if defined(RCEXPLORERF3)
```

Comment:
Not overlooked. When I did the original refactoring out of code into `target.c` I left this bit out because it was very complicated and refactoring it had a high risk of accidentally breaking targets. The intention was to revisit it later.

@blckmn has been doing further work in this area, including the ability to assign pins at configuration time (rather than compile time) - this work is planned for 3.1